lms/resources/_js_config/__init__.py

```diff
@@ -329,6 +329,13 @@ def _add_canvas_speedgrader_settings(self, **kwargs):
                 **kwargs,
             },
         }
+        # Enable the LMS frontend to receive notifications on annotation activity
+        # We'll use this information to only send the submission to canvas on first annotation.
+        if self._request.feature("submit_on_annotation"):
```

Comment:
```suggestion
        if self._request.feature("submit_on_annotation"):
            # The `reportActivity` setting causes the front-end to make a call 
            # back to the parent iframe for the specified events. The method in 
            # the iframe happens to be called `reportActivity` too, but this is
            # a co-incidence. It could have any name.
```